python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1362,3 +1374,327 @@ def detectClipBig(self, tempExpClipList, clipFootprints, clipIndices, maskClipVa
             tmpExpMask |= maskVisitClip
 
         return bigFootprintsCoadd
+
+
+class CompareWarpAssembleCoaddConfig(AssembleCoaddConfig):
+    assembleStaticSkyModel = pexConfig.ConfigurableField(
+        target=AssembleCoaddTask,
+        doc="Task to assemble clean, artifact-free, PSF-matched Coadd to serve as model of "
+            "the static sky",
+    )
+    chiThreshold = pexConfig.Field(
+        doc="Pixels in the chi-image of warp minus the model of the static sky (template coadd)",
```

Comment:
I don't really know what this means. Could you re-word or add to this description?